### PR TITLE
Display menu mobile

### PR DESCRIPTION
**Pull request by SamuelTTW**
Add animation to the hamburger Button in the style.css file and add functionality in main.js file 

Mobile version display now the menu

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const lBars1 = document.querySelector('.line1__bars-menu');
+const lBars2 = document.querySelector('.line2__bars-menu');
+const lBars3 = document.querySelector('.line3__bars-menu');
+const listBars = document.querySelector('.list_bar-menu');
+
+document.querySelector('#bmenu').addEventListener('click', () => {
+  lBars1.classList.toggle('activeline1__bars-menu');
+  lBars2.classList.toggle('activeline2__bars-menu');
+  lBars3.classList.toggle('activeline3__bars-menu');
+  listBars.toggle('deploy');
```
Comment (ishimwezachee) [inline js/main.js]: - [ ]  You javascript has some issue that you need to fix. When I click on your humburg menu, I  am getting an error in in console. and this is blocking your navigations for small device to appear. 

- [ ]  You should use same cross `x`similar to the one from figma design. 
image

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+});
+document.querySelector('.a2').addEventListener('click', () => {
+  lBars1.classList.toggle('activeline1__bars-menu');
+  lBars2.classList.toggle('activeline2__bars-menu');
+  lBars3.classList.toggle('activeline3__bars-menu');
+  listBars.toggle('deploy');
+});
```
Comment (ishimwezachee) [inline js/main.js]: - [ ] Fix The issue and make sure that you project has fulfilled the following ...

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappear.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- When the user clicks (or taps) any option of the mobile menu, the mobile menu disappear.
- The hamburger button and menu follows the design from [the chosen template in Figma]

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @SamuelTTW and @looh16 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Ishimwe Zachee  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(Not automatically expanded because 3 MB is too large. You can [expand it anyway](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) or [open it in a new window](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966).)

File: index.html [hunk 3]
```diff
@@ -19,27 +19,48 @@
...
+          <a class="a0" href="#main-container">Portfolio</a>
+        </li>
+        <li>
+          <a class="a1" href="#about-me">About</a>
+        </li>
+        <li>
+          <a class="a2" href="#footer">contact</a>
+        </li>
+      </ul>
+    </div>
```
Comment (Mre55) [inline index.html:63]: Let's stick to the provided design. The `mobile menu` should have to cover the full screen and it shouldn't have to be scrollable. The only way the user can see the other content of the page is by either choosing one of the options or closing the `mobile menu`. Kindly fix that.

https://user-images.githubusercontent.com/34578631/174819273-2f1973bd-6b51-4927-8879-5192b6e26668.mov

File: index.html [hunk 4]
```diff
@@ -19,27 +19,48 @@
  
     <header>
       <!-- MOBILE-&-TABLET-VERSION -->
-      <a href="">Welcome</a>
+      <a class="welcome" href="">Welcome</a>
```
Comment (Mre55) [inline index.html:22]: When the `mobile menu` is shown to the user everything else should be in the back and hidden. Kindly, remove the `Welcome` text and some parts of the home page when the `mobile menu` is selected. Also, give some padding to the left of the text and padding of right for the `X` icon to make it more similar to the design.

| Your Design | Figma Design |
|-------------|---------------|
| image | image |

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @SamuelTTW,**

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

 ## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions 🟡 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @SamuelTTW,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**